components/constellation/browsingcontext.rs

```diff
@@ -15,15 +15,8 @@ use style_traits::CSSPixel;
 /// context which doesn't exist yet, these values needed for the new browsing
 /// context are stored here so that they may be available later.
 pub struct NewBrowsingContextInfo {
-    /// The parent pipeline of the browsing context, if this is not the top
-    /// level browsing context.
```

Comment:
Why remove these doc comments?

Reply:
I thought they were redundant since the same fields with docs can be found a few lines below. I can add them back if you think redundancy in this case is preferred.

Reply:
@mandreyel This is a public interface, so if you go to doc.servo.org and look for `NewBrowsingContextInfo` once this PR is merged, you should see documentation comments on these 3 fields. I do not expect people to necessarily look at the documentation for `BrowsingContext` when what they need is `NewBrowsingContextInfo`, so please add them back.

Reply:
@KiChjang Ah, I hadn't thought of that! That makes sense, I'll change it back then. Thanks!